Cargo.toml

```diff
@@ -40,9 +40,11 @@ relay-general = { path = "relay-general" }
 sentry = { version = "0.18.0", features = ["with_debug_meta"] }
 serde = "1.0.98"
 serde_json = "1.0.40"
+serde_yaml = "0.8.9"
 
 [target."cfg(not(windows))".dependencies]
 openssl-probe = "0.1.2"
 
 [build-dependencies]
 clap = { version = "2.33.0", default-features = false }
+insta = { version = "0.12.0", features =  ["ron", "redactions"] }
```

Comment:
Can you back out the changes to `/Cargo.toml`? `serde_yaml` is already a dependency of `relay-config`.

If you need `insta`, please use version `0.15.0` which is used in other crates, and move the dependency to `relay-config/Cargo.toml`.